src/components/Callout/Callout.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { CalloutProps } from './Callout.types';
+import { ColorTypes, SpaceSizes } from '../../theme';
+import { getColor, getFontSize, getRadii, pxToRem } from '../../utils';
+import { Inline, Padbox } from '../layout';
+import { Text } from '../typographyLegacy';
+import { Icon } from '../Icon';
+
+const IconContainer = styled.div`
+  width: ${pxToRem(36)};
+  height: ${pxToRem(36)};
+  flex-shrink: 0;
+  background-color: ${getColor(ColorTypes.info700)};
+  border-radius: ${getRadii('circle')};
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: ${getFontSize('mdPlus')};
+`;
+
+const Container = styled.div`
```

Comment:
you can change `div` to `Padbox` and add the padding size to the Container component and save one DOM node.